text/xxxx-make-KLT-standard-for-new.users.md

```diff
@@ -0,0 +1,38 @@
+# Make Keptn Lifecycle Toolkit the standard for new users
+
+KLT is the next step after Keptn V 1.x. It is more flexible, better integrated into Kubernetes and also more lightweight ean extensible. Therefore we should direct new user to start their journey with the lifecycle toolkit right away.
+
+## Motivation
+
+* New users get started on the latest version of Keptn
+* New extensions/services will also be built in a future proof way
+* We avoid that users later have to migrate
+* We reduce confusion between the lifecycle toolkit and Keptn V 1.x
+* This will allow us to focus any new developments on the lifecycle toolkits
+
+## Explanation
+
+* We rework the website to be better alinged with the lifecycle toolkit.
+* All onboarding flows will be directed towards the lifecycle toolkit an lifecycle toolkit docs. 
+* The current Keptn V1.x documentation will still be available but under a dedicated menu item.
+* All V1.x ressources will have a banner stating "If you are new to Keptn we recomment already starting with the lfiecycle toolkit"
+
+## Internal details
+
+
+## Trade-offs and mitigations
+
+* Existing users might be confused when coming to the Keptn website. *Mitigation* we have been informing everybody about our plans sie last fall
```

Comment:
```suggestion
* Existing users might be confused when coming to the Keptn website. *Mitigation* we have been informing everybody about our plans since last fall
```